main/como/knock_out_simulation.py

```diff
@@ -123,46 +124,27 @@ def knock_out_simulation(
     
     # Require at least one core
     num_cores: int = max(1, mp.cpu_count() - 2)
```

Comment:
We're determining the maximum number of cores here. Should this instead be a value passed in through the command line?

Reply:
This is a valid approach. Because COMO is running in a container, we can limit the number of cores available without impacting the host